### DISCUSSION TITLE
Heroku two factor authentication

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#1952] [Show and tell]
If you use Nightscout, you should enable two factor authentication in your Herku account before February.

https://nightscout.github.io/vendors/heroku/

https://docs.google.com/document/d/1ZpoCIPwBmpm47MJwkY5wXTmnEtrtqYRpfaO1oNl29_E/edit?fbclid=IwAR0yiUOhC3sGQYdjKsxTO1uAm1qtbtBeL2w4x7j-9ajQuUNsgystIx_8Nks

**Comment by psonnera** [comment]
Also here https://nightscout.github.io/vendors/heroku/ comments and PRs welcome.

**Comment by tolot27** [comment]
Does it have any impact on xDrip?

**Reply by Navid200** [reply]
Would you please unpin this discussion?

**Comment by psonnera** [comment]
No. Only Heroku.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
